### ISSUE TITLE
it doesn't work for latest version of AL2023

### ISSUE THREAD

**Issue by bryanfang** [livingstaccato/freeipa-amzn2023#1]
Hi @livingstaccato,

I am trying to install FreeIPA to the latest AL2023 AMI, but it doesn't work now because of dependencies issue, I guess the SPEC files need to be adjusted, could you please check once when you get a chance, thanks a lot!

`Last metadata expiration check: 0:05:26 ago on Thu Dec 28 01:56:23 2023.
Error:
 Problem 1: conflicting requests
  - nothing provides python3-ldap >= 3.1.0-1 needed by freeipa-4.11.0-7.amzn2023.src
 Problem 2: conflicting requests
  - nothing provides (python3dist(wrapt) < 2~~ with python3dist(wrapt) >= 1.10) needed by python-deprecated-1.2.13-4.amzn2023.src
 Problem 3: conflicting requests
  - nothing provides python3dist(pypng) needed by python-qrcode-7.4.2-7.amzn2023.src
 Problem 4: conflicting requests
  - nothing provides python3-pyusb needed by python-yubico-1.3.3-13.amzn2023.src
(try to add '--skip-broken' to skip uninstallable packages)`

BR,
Bryan

**Comment by livingstaccato** [comment]
@bryanfang Yeah - what I've got is very rudimentary so far. I'll have something that builds using docker soon up… Stay tuned :)

**Comment by livingstaccato** [comment]
Hey there - I'm setting something up to host Amazon Linux packages at https://repo.l12o.org/rpm/amzn/2023/ . I'm still working on getting some CI/CD stuff setup and package signing
